<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tag editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="fileE"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Przeglądaj</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="oocFB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Plik</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="updateB">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Uaktualnij</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="tagGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>231</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Enabled</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="titleE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="artistE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="albumE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="commentE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextEditorInteraction</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="yearB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="trackB">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="genreE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="clrB">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Wyczyść</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="pictureGroupBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>130</width>
        <height>150</height>
       </rect>
      </property>
      <property name="title">
       <string>Picture</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileE</tabstop>
  <tabstop>browseB</tabstop>
  <tabstop>oocFB</tabstop>
  <tabstop>tagGroupBox</tabstop>
  <tabstop>titleE</tabstop>
  <tabstop>artistE</tabstop>
  <tabstop>albumE</tabstop>
  <tabstop>commentE</tabstop>
  <tabstop>genreE</tabstop>
  <tabstop>yearB</tabstop>
  <tabstop>trackB</tabstop>
  <tabstop>pictureGroupBox</tabstop>
  <tabstop>updateB</tabstop>
  <tabstop>clrB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
